Agregado seleccionar plan de estudios

eanorambuena opened the pull request:
#1

eanorambuena committed:
Agregado seleccionar plan de estudios

Dyotson commented:
El problema de esta solucion es que el Selenium abre una pantalla completa, por lo cual el usuario no podria ver que tiene que ingresar su plan de estudios, hay que buscar la forma que sea mas rapido y sencillo. Ademas estaria bueno testearlo antes de agregarlo al main

eanorambuena commented:
La toma de ramos ya fue, ¿cómo lo han estado testeando? Ningún problema en hacerlo.
En cuanto a la pantalla completa se podría entonces informar el plan de estudios al inicio con lo demás y hacer una búsqueda por subcadenas que no debería agregar un tiempo notorio.

Dyotson commented:
Me tinca esa idea, ¿Podriamos implementarla? Lo he estado testeando con diferentes personas dispuestas a sacrificar un poco de su tiempo, aún tengo gente por si quieres testear

eanorambuena commented:
Claro!

eanorambuena committed:
Modificado selector de plan de estudios

Ya no está en el main.
Se usa búsqueda por subcadenas ~O(n) insignificante para este caso

eanorambuena commented:
Puse una búsqueda de subcadenas básica. Obvio abierto a sugerencias/tests :D

eanorambuena committed:
Bug auto-solucionado

eanorambuena committed:
Update README.md

eanorambuena committed:
Merge branch 'main' into Readme

Dyotson commented:
Intente ejecutar tu fork, me dio el siguiente error:
`  File "main.py", line 95
    if normalize(planseleccionado.lower()) in normalize(planes[i].lower()) # Quita tildes y pone ambos strings en minúsculas para que sea la búsqueda sea insensible a variaciones
                                                                           ^
SyntaxError: invalid syntax`

eanorambuena commented:
Cuando comenté quité el ":", debe ser eso
